addons/mail/static/tests/mock_server/mail_mock_server.js

```diff
@@ -9,55 +17,167 @@ import { registry } from "@web/core/registry";
 
 const { DateTime } = luxon;
 
-//-----------------------------------------------------------------------------
-// Internal
-//-----------------------------------------------------------------------------
+/**
+ * @param {Array} param arguments of method
+ * @param  {...string} argNames ordered names of positional arguments
+ * @returns {Object} kwargs normalized params
+ */
+export const parseModelParams = (params, ...argNames) => {
```

Comment:
I just use this in mail mock model code. This function is how I solve the args/kwargs stuff. I'm not entirely sure this approach is ok for everyone. I put things in `mail` helpers when I could, so that we need it but other don't necessarily need it.

Probably could be in `web` if we think it's worth it being there eventually